src/style.css

```diff
@@ -1,3 +1,133 @@
+:root {
+  --link-color: #00acdc;
+}
+
+html {
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+  font-weight: 500;
+}
+
 body {
   margin: 0;
 }
+
+.header {
+  width: 100%;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding: 0 50px;
+  margin-top: 10px;
+  margin-bottom: 10px;
+  height: 60px;
+  box-sizing: border-box;
+}
+
+.nav-logo a {
+  display: flex;
+  align-items: center;
+}
+
+img {
+  width: 40px;
+  height: 40px;
+}
+
+.nav {
+  display: flex;
+  align-items: center;
+}
+
+.nav-links {
+  display: flex;
+  justify-content: flex-end;
+}
+
+ul {
+  list-style: none;
+  padding: 0;
+  margin: 0;
+  line-height: 60px;
+}
+
+li {
+  display: inline-block;
+  margin-left: 17px;
+  text-align: center;
+  height: 60px;
+  line-height: 60px;
+}
+
+li:first-child {
+  margin-left: 0;
+}
+
+.nav a {
+  all: unset;
+  text-decoration: none;
+  cursor: pointer;
+  display: flex;
+}
+
+a[data-qa='hover'] {
```

Comment:
The 'a[data-qa='hover']' selector is empty and should either be removed or filled with relevant styles.